src/internal/Subscriber.ts

```diff
@@ -106,6 +106,7 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
     if (!this.closed) {
       this.isStopped = true;
       super.unsubscribe();
+      this.destination = NOOP_OBSERVER;
```

Comment:
at the point we've unsubscribed, it seems like we could just set `this.destination = null!` here, although I haven't tried that and tested.

Reply:
I believe simply nulling it out caused several tests to fail.